core/modules/admin_bar/js/admin_bar.admin.js

```diff
@@ -18,14 +18,14 @@ Backdrop.behaviors.adminBarPermissionsSetupHelp = {
       var $menu = $(this).find('input[name$="[access administration bar]"]');
 
       // Retrieve the permission label - without description.
-      var adminPermission = $.trim($admin.eq(0).parents('td').prev().children().get(0).firstChild.textContent);
-      var menuPermission = $.trim($menu.eq(0).parents('td').prev().children().get(0).firstChild.textContent);
+      var adminPermission = $admin.eq(0).parents('td').prev().children().get(0).firstChild.textContent.trim();
+      var menuPermission = $menu.eq(0).parents('td').prev().children().get(0).firstChild.textContent.trim();
```

Comment:
These two `.trim()` calls look fine since textContent should always be a string.